projects/frontegg-app/src/lib/frontegg-app.service.ts

```diff
@@ -178,4 +183,12 @@ export class FronteggAppService {
   public hideCheckoutDialog(): void {
     this.fronteggApp?.hideCheckoutDialog();
   }
+
+  /**
+    @param keys The requested entitlement keys
+    @returns Entitlements contain true/false for every key (state of is key entitled)
+  */
+  public getEntitlements(keys: string[]): Entitlements {
```

Comment:
maybe more declarative name then keys?

Reply:
this is the terminology we use also in the BE. I wanna be aligned